ui-tests/src/test/java/org/openmrs/reference/CheckModulesTest.java

```diff
@@ -1,29 +1,30 @@
 package org.openmrs.reference;
 
+import java.util.List;
+
 import org.junit.Assert;
-import org.junit.Ignore;
 import org.junit.experimental.categories.Category;
+import org.junit.jupiter.api.Test;
+import org.openmrs.reference.groups.BuildTests;
 import org.openmrs.reference.page.AdministrationPage;
 import org.openmrs.reference.page.ModulesPage;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
 
-import java.util.List;
-
 public class CheckModulesTest extends ReferenceApplicationTestBase {
 
     /**
      * Check the list of modules to be sure they are all started.
      */
-    @Ignore
-    @Category(org.openmrs.reference.groups.BuildTests.class)
-    public void checkModules() throws Exception {
+    @Test
+    @Category(BuildTests.class)
+    public void checkModulesTest() {
         AdministrationPage administrationPage = homePage.goToAdministration();
         ModulesPage modulesPage = administrationPage.goToManageModulesPage();
         // Get the modulesListing <div>, which contains the table of modules.
         WebElement moduleListing = modulesPage.findElementById("moduleListing");
         // Grab all the <input> elements from the first column of the table.
-        List<WebElement> firstColumn = moduleListing.findElements(By.xpath("table/tbody/tr/td[1]/input"));
+        List<WebElement> firstColumn = moduleListing.findElements(By.cssSelector("#moduleListing table tbody td"));       
```

Comment:
```suggestion
Why not using this as an id , something like 
        List<WebElement> firstColumn = moduleListing.findElements(By.cssSelector("#moduleListing"));       
```

Reply:
I gave on attempt using id locator but test fail sand the one element locator I found out passing the test was by using `cssSelector()`